src/golang/cmd/server/handler/list_artifact_results.go

```diff
@@ -0,0 +1,146 @@
+package handler
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/aqueducthq/aqueduct/cmd/server/routes"
+	db_artifact "github.com/aqueducthq/aqueduct/lib/collections/artifact"
+	"github.com/aqueducthq/aqueduct/lib/collections/artifact_result"
+	"github.com/aqueducthq/aqueduct/lib/collections/workflow_dag"
+	aq_context "github.com/aqueducthq/aqueduct/lib/context"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/storage"
+	"github.com/aqueducthq/aqueduct/lib/workflow/artifact"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+)
+
+// Route: /artifact/{artifactId}/results
+// Method: GET
+// Params: None
+// Request:
+//	Headers:
+//		`api-key`: user's API Key
+// Response:
+//	Body:
+//		serialized `listArtifactResultsResponse`
+
+type listArtifactResultsResponse struct {
+	Results []artifact.RawResultResponse `json:"results"`
```

Comment:
Can you note what order these results are returned in? Is it last executed first?

Reply:
there's no order assumption here, we should handle that on client side

Reply:
how does the client order it? Using the timestamps field in ExecState?

Reply:
yes

Reply:
Can we just add a comment here to that effect?